--- a/tests/cavern.tst
+++ b/tests/cavern.tst
@@ -105,7 +105,7 @@
  samename tabinhighlight legacytokens\
  3dexport \
  dxffullcoords dxfsurfequate\
- gpxexport hpglexport jsonexport kmlexport pltexport svgexport\
+ gpxexport jsonexport kmlexport pltexport svgexport\
 "}}
 
 # Test file stnsurvey3.svx missing: pos=fail # We exit before the error count.
@@ -342,6 +342,11 @@
 	sed 's/-0\.00\>/ 0.00/g' < "$tmpfile" > tmp.tmp
 	mv tmp.tmp "$tmpfile"
 	;;
+      json)
+	# On x86 excess precision can result in -0.00 for some coordinates.
+	sed 's/-0\.00\>/0.00/g' < "$tmpfile" > tmp.tmp
+	mv tmp.tmp "$tmpfile"
+	;;
       gpx)
 	sed 's,<time>[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9]Z</time>,<time>REDACTED</time>,;s,survex [0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*,survex REDACTED,' < "$tmpfile" > tmp.tmp
 	mv tmp.tmp "$tmpfile"
