Description: Fix wx assertion
Author: Olly Betts <olly@survex.com>
Forwarded: in-progress
Last-Update: 2018-07-03

--- survex-1.2.35.orig/src/aventreectrl.cc
+++ survex-1.2.35/src/aventreectrl.cc
@@ -354,11 +354,11 @@ void AvenTreeCtrl::OnMenu(wxTreeEvent& e
 		menu.Enable(menu_SURVEY_HIDE, false);
 		menu.Enable(menu_SURVEY_HIDE_SIBLINGS, false);
 		break;
-#endif
 	    case STATE_NONE:
 		menu.Enable(menu_SURVEY_HIDE, false);
 		menu.Enable(menu_SURVEY_HIDE_SIBLINGS, false);
 		break;
+#endif
 	}
 	PopupMenu(&menu);
     }
